frame.py

```diff
@@ -0,0 +1,34 @@
+import os
+import urllib2
+from const import SUPPORTED_FILE_TYPES
+
+HASH_NOTEPAD = str(os.environ.get("HASH_NOTEPAD"))
+HASH_PAINT = str(os.environ.get("HASH_PAINT"))
+
+
+def file_exist(file_url):
+    """
+    Method that check if file at provided url exist.
+    :param file_url: Url of the file that should be checked.
+    :return: True if file exist, otherwise False.
+    """
+    request = urllib2.Request(file_url)
```

Comment:
I suggest you to use library `requests`. It has much better interface for making requests and it retries by default in case network errors, timeouts, etc.